app/views/layouts/application.html.haml

```diff
@@ -23,7 +25,7 @@
         .wrappable__content.wrappable__content--flex.main-pane
           %main
             = render 'application/breadcrumbs'
-            = yield
+            = content_for?(:content) ? yield(:content) : yield
```

Comment:
see for detail http://guides.rubyonrails.org/layouts_and_rendering.html#using-nested-layouts